lib/ui/notifications.dart

```diff
@@ -87,39 +88,88 @@ class NotificationsWidgetState extends AssistantState<NotificationsWidget> {
             ),
             Tab(
               text: "Manage",
-            )
+            ),
           ]),
         ),
         body: TabBarView(
           children: <Widget>[
-            Container(
+            Container( //Notifications
               child: Column(
                 children: [
                   Row(
                       mainAxisAlignment: MainAxisAlignment.spaceAround,
                       children: [
                         Container(
-                            child: Text('Date'),
+                            child: Text('Date',style:TextStyle(color:Color.fromRGBO(51, 51, 102, 1),
+                                fontSize: 18),),
                             padding:
                                 EdgeInsets.only(left: 40, top: 20, bottom: 5)),
                         Container(
-                          child: Text('Keyword(s)'),
+                          child: Text('Keyword(s)',style:TextStyle(color:Color.fromRGBO(51, 51, 102, 1),
+                              fontSize: 18),),
                           padding:
```

Comment:
Do you intend to add a manage button or something? currently it appears the delete button aligns with Keyword(s) and actual keyword aligns with Date 

![image](https://user-images.githubusercontent.com/4079954/196471956-63ea32f9-4781-4f0c-a336-49b32348f61e.png)

Reply:
The mockup design didn't have a "manage/edit" button but I can definitely add it. Also, date has been removed from the Notification object so do we still need that field? If we do, we'll need to add that field to the database.

I'll continue working on the cleaning up the UI for this Notifications tab.

Reply:
Yeah we'll want the timestamp of the notification

Reply:
I believe Natan was working on the notifications page, so added the code I used on the manage notifications tab merely for demonstration purposes.  I left the following comment there for him to review and to be replaced once he was ready with his own code.
//the following code is used for notification subscriptions on the manage tab, but placed here for testing purposes and layout